applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaSphereModel.h

```diff
@@ -35,8 +35,8 @@ namespace gpu
 namespace cuda
 {
 
-typedef sofa::component::collision::SphereCollisionModel<gpu::cuda::CudaVec3Types> CudaSphereModel;
-typedef sofa::component::collision::TSphere<gpu::cuda::CudaVec3Types> CudaSphere;
+using CudaSphereModel [[deprecated("The CudaSphereModel is now deprecated, please use sofa::component::collision::SphereCollisionModel<gpu::cuda::CudaVec3Types> instead. Compatibility stops at v20.06")]] = sofa::component::collision::SphereCollisionModel<gpu::cuda::CudaVec3Types>;
```

Comment:
I'm not a big fan of sofa::component::collision.... so I suggest to do
```cpp
using sofa::component::collision::SphereCollisionModel
using gpu::cuda::CudaVec3Types; 
```
So we can then use 
```cpp
SphereCollisionModel<CudaVec3Types> ... 
```

Reply:
Dans fichier toto.h
```cpp
namespace titi
{
    namespace _toto_
    {
        class Tata {};
    }
    using _toto_::Tata;
}
```